File: Java/01.Java-Basic/02.FirstStepsInCodingExercise/src/FruitMarket.java
import java.util.Scanner;

public class FruitMarket {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double yagodiPrice = Double.parseDouble(scanner.nextLine());
        double banani = Double.parseDouble(scanner.nextLine());
        double portokali = Double.parseDouble(scanner.nextLine());
        double malini = Double.parseDouble(scanner.nextLine());
        double yagoddi = Double.parseDouble(scanner.nextLine());

        double maliniPrice = yagodiPrice / 2;
        double portokaliPrice = maliniPrice - (maliniPrice * 0.4);
        double bananiPrice = maliniPrice - (maliniPrice * 0.8);

        double yagodiTotalPrice = yagoddi * yagodiPrice;
        double maliniTotalPrice = malini * maliniPrice;
        double bananiTotalPrice = banani * bananiPrice;
        double portokaliTotalPrice = portokali * portokaliPrice;
        double totalPrice = yagodiTotalPrice + maliniTotalPrice + bananiTotalPrice + portokaliTotalPrice;
        System.out.printf("%.2f", totalPrice);
    }
}

File: JavaScript/01.JavaScriptBasic/02FirstStepsExercise/FruitMarket.js
function solve(input) {
    let yagodiPrice = Number(input[0]);
    let banani = Number(input[1]);
    let portokali = Number(input[2]);
    let malini = Number(input[3]);
    let yagodi = Number(input[4]);
    
    let maliniPrice = yagodiPrice / 2;
    let portokaliPrice = maliniPrice - (maliniPrice * 0.4);
    let bananiPrice = maliniPrice - (maliniPrice * 0.8); 
    let sum = (yagodiPrice * yagodi) + (malini * maliniPrice) + (portokali * portokaliPrice) + (banani * bananiPrice);
    console.log(sum);
}

solve(["48",
"10",
"3.3",
"6.5",
"1.7"]);

solve(["63.5",
"3.57",
"6.35",
"8.15",
"2.5"]);